ocaml/xenops/domain.ml

```diff
@@ -211,14 +208,12 @@ let make ~xc ~xs vm_info uuid =
 			(* create cpu and memory directory with read only perms *)
 			List.iter (fun dir ->
 				let ent = sprintf "%s/%s" dom_path dir in
-				t.Xst.mkdir ent;
-				t.Xst.setperms ent roperm
+				t.Xst.mkdirperms ent roperm
 			) [ "cpu"; "memory" ];
 			(* create read/write nodes for the guest to use *)
 			List.iter (fun dir ->
 				let ent = sprintf "%s/%s" dom_path dir in
-				t.Xst.mkdir ent;
-				t.Xst.setperms ent rwperm
+				t.Xst.mkdirperms ent rwperm
 			) [ "device"; "error"; "drivers"; "control"; "attr"; "data"; "messages"; "vm-data"; "hvmloader" ];
 		);
```

Comment:
Do you need to add the `xenops_dom_path` as in https://github.com/xapi-project/xenopsd/pull/270/files#diff-356aa9d32f55797721b171382715f41fR264?